includes/class-wc-payments-account.php

```diff
@@ -1030,6 +1055,10 @@ public function maybe_handle_onboarding() {
 		// Determine the original/initial place from where the merchant entered the onboarding flow.
 		$onboarding_source = WC_Payments_Onboarding_Service::get_source();
 
+		// Determine if the user is part of an active experiment variation (aka treatment).
+		$abtest     = ! empty( $_GET['abt'] ) ? sanitize_text_field( wp_unslash( $_GET['abt'] ) ) : false;
```

Comment:
why not use `WC_Payments_Utils::is_in_core_payments_task_onboarding_flow_treatment_mode()` to determine if the user is part of an active experiment variation ?